### PR TITLE
Stats/Prometheus: export data using metrics API

### PR DESCRIPTION
**Pull request by mayurkale22**
This PR is part of #1483

### CODE REVIEW

File: exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java [hunk 1]
```diff
@@ -222,59 +200,37 @@ public Void apply(DistributionData arg) {
...
                   new MetricFamilySamples.Sample(
-                      name + SAMPLE_SUFFIX_SUM,
-                      labelNames,
-                      labelValues,
-                      meanData.getCount() * meanData.getMean()));
-              return null;
+                      name + SAMPLE_SUFFIX_SUM, labelNames, labelValues, sum));
             }
-            throw new IllegalArgumentException("Unknown Aggregation.");
+            return samples;
```
Comment (songy23) [inline exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java]: How about percentiles?

File: exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java [hunk 2]
```diff
@@ -88,127 +74,119 @@
...
-        name, type, view.getDescription(), Collections.<Sample>emptyList());
+        name, type, metricDescriptor.getDescription(), Collections.<Sample>emptyList());
   }
 
   @VisibleForTesting
-  static Type getType(Aggregation aggregation, View.AggregationWindow window) {
-    if (!(window instanceof View.AggregationWindow.Cumulative)) {
-      return Type.UNTYPED;
+  static Type getType(MetricDescriptor.Type type) {
+    if (type == MetricDescriptor.Type.CUMULATIVE_INT64
```
Comment (songy23) [inline exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java]: `CUMULATIVE_INT64` and `CUMULATIVE_DOUBLE` can also represent cumulative sum, so the values could increment by more than 1 (e.g sum 10 -> 20 -> 30). Does Prometheus Counter allow that?

File: exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java [hunk 3]
```diff
@@ -23,59 +23,45 @@
...
- * DistributionData} will be converted to a list of {@link Sample}s that have the sum, count and
- * histogram buckets.
- *
- * <p>{@link TagKey} and {@link TagValue} will be converted to Prometheus {@code LabelName} and
- * {@code LabelValue}. {@code Null} {@link TagValue} will be converted to an empty string.
+ * <p>{@link LabelKey} and {@link LabelValue} will be converted to Prometheus {@code LabelName} and
+ * {@code LabelValue}. {@code Null} {@link LabelValue} will be converted to an empty string.
  *
  * <p>Please note that Prometheus Metric and Label name can only have alphanumeric characters and
  * underscore. All other characters will be sanitized by underscores.
```
[the remark this answers is not in the document]
Comment (mayurkale22) [reply exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java]: Done.

File: exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java (hunk 1 above)
Comment (mayurkale22) [reply exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java]: Added Summary Percentiles.

File: exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java (hunk 2 above)
Comment (mayurkale22) [reply exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java]: FYI https://github.com/prometheus/client_java/blob/master/simpleclient/src/main/java/io/prometheus/client/Counter.java#L117-L126

**Pull request merged**